src/Behat/Transliterator/Transliterator.php

```diff
@@ -573,7 +573,10 @@ private static function postProcessText($text, $separator)
             $text = strtolower($text);
         }
 
-        // Remove all none word characters
+        // Remove apostrophes which are not used as quotes around a string
+        $text = preg_replace('|(\\w)(\')(\\w)|', '${1}${3}', $text);
```

Comment:
If I were you, I wouldn't use `|` as regex delimiter (not particularly here, but on a global scale), as these can be pretty common. Use `/`s instead (or better, `{...}`), as it is used in the lib.

And btw, you don't need to escape the `\`s